src/Photos/Enums.cs

```diff
@@ -378,12 +382,26 @@ public enum FigExifCustomRenderedValue : short {
 	[TV (13,0), Mac (10,15), iOS (13,0)]
 	[Native]
 	public enum PHPhotosError : long {
+		[Deprecated (PlatformName.MacOSX, 12, 0, message: "Use 'InternalError' instead.")]
+#if !XAMCORE_4_0
 		Invalid = -1,
+#endif
+		InternalError = -1,
```

Comment:
Will having 2 enums with the value -1 cause issues?

Reply:
Nope, it's totally valid to have multiple enum fields with the same numerical value.

Reply:
Oh wait, the [Deprecated] attribute goes inside the #if as well.